contrib/refresh_checksums.mk

```diff
@@ -47,7 +47,7 @@ endef
 # if $(3) is "assert", we set BINARYBUILDER_LLVM_ASSERTS=1
 define checksum_dep
 checksum-$(1)-$(2)-$(3):
-	-$(MAKE) $(QUIET_MAKE) -C "$(JULIAHOME)/deps" $(call make_flags,$(1),$(2),$(3)) checksum-$(1)
+	-+$(MAKE) $(QUIET_MAKE) -C "$(JULIAHOME)/deps" $(call make_flags,$(1),$(2),$(3)) checksum-$(1)
```

Comment:
I'm not entirely sure what the `+` prefix here means; is this intended to print the command but not actually execute it?

Reply:
It tells make that the program executed on this line is `make`. It is often auto-detected, but seemed to be broken here